.rubocop.yml

```diff
@@ -3,6 +3,9 @@ require:
 
 AllCops:
   NewCops: enable
+  Include:
+    - Guardfile
+    - Rakefile  
```

Comment:
Is this really needed?

Reply:
they are not necessary, only if you want to customize them. Removing.